src/etc/inc/system.inc

```diff
@@ -73,7 +73,9 @@ function system_sysctl_defaults()
         'debug.kassert.warn_only' => [ 'default' => '1', 'required' => true, 'description' => 'KASSERT triggers a panic (0) or just a warning (1)', 'type' => 'w' ],
         'hw.ibrs_disable' => [ 'default' => '0' ],
         'hw.ixl.enable_head_writeback' => [ 'default' => '0', 'required' => true ],
+        'hw.vtnet.csum_disable' => [ 'default' => '1', 'required' => true ],
```

Comment:
you really only need this line

I'm not overly enthusiastic about having two spots and the default already being disabled, but having an automatic sysctl would be ok

Reply:
> I'm not overly enthusiastic about having two spots and the default already being disabled, but having an automatic sysctl would be ok

The default of hardware offloading is enabled - "disabled" sysctl == 0 - I am suggesting to disable it by default for new OPNsense installs. Reason being that it doesn't hurt but helps a lot with the majority of common cloud providers.